bfd/src/main/java/gov/cms/ab2d/bfd/client/BFDSearchImpl.java

```diff
@@ -79,6 +80,17 @@ public IBaseBundle searchEOB(long patientId, OffsetDateTime since, int pageSize,
 
     @Trace
     private IBaseBundle parseBundle(FhirVersion version, byte[] responseBytes) {
+
```

Comment:
Please add a TODO with explanation as to why commented out code is being added to the system.